rubocop.gemspec

```
@@ -39,7 +39,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rainbow', '>= 2.2.2', '< 4.0')
   s.add_runtime_dependency('rexml')
   s.add_runtime_dependency('ruby-progressbar', '~> 1.7')
-  s.add_runtime_dependency('unicode-display_width', '>= 1.4.0', '< 1.7')
+  s.add_runtime_dependency('unicode-display_width', '>= 1.4.0', '< 1.8')
```

I think it can specify a pre-released major version 2.0.
https://rubygems.org/gems/unicode-display_width

```suggestion
  s.add_runtime_dependency('unicode-display_width', '>= 1.4.0', '< 2.0')
```